can-view-parser.js

```diff
@@ -307,6 +307,21 @@ var camelCaseToSpinalCase = function (match, lowerCaseChar, upperCaseChar) {
 	return lowerCaseChar + "-" + upperCaseChar.toLowerCase();
 };
 
+HTMLParser.sanitizeAttrs = function(html) {
+	// represent:
+	// <tag attrs="special characters that can muck up parsing"></tag>
+	// as:
+	// <tag attrs=.............................................></tag>
+	//
+	// regex explanation:
+	// (['"]) capture an opening ' or "
+	// (?:(?!\1).)* match zero or more characters that aren't the captured ' or "
+	// \1 match the closing ' or "
+	return html.replace(/(['"])(?:(?!\1).)*\1/g, function(match) {
```

Comment:
- `<span>'</span><span attr='>'></span>` does break things. `sanitizeAttr()` is only properly sanitizing the first tag. It's luck that by the time `searchStartTag()` calls again, user content has been dealt with.
- escaped quotes in attributes is a bigger deal and have to be properly handled
- I don't think line breaks require special treatment, here, unless you can come up with a breaking test :)